static-site/content/featured-analyses.yaml

```diff
@@ -4,16 +4,16 @@
 # - When adding or modifying descriptions, ensure it takes up at most two lines
 #   when rendered on the splash page.
 
+- name: HPAI outbreaks
+  description: "HPAI outbreak dynamics in North America: 2021-present"
```

Comment:
_nit_

Is it HPAI outbreak**s** or HPAI outbreak?

Reply:
That's copypastaed from [the narrative title](https://next.nextstrain.org/community/narratives/moncla-lab/nextstrain-narrative-hpai-north-america@main/HPAI-in-North-America) so I think it's probably correct?

Reply:
The description "HPAI outbreak dynamics" is from the narrative title, but what about the name "HPAI outbreaks" here? The plurality is not consistent.

Reply:
Thanks guys! To be honest, this hasn't been updated for awhile. Maybe we should change to 2021-2023? It doesn't include any cattle data or D1.1 emergence, so isn't particularly timely/up to date.

Reply:
@victorlin switched card title to the singular.